config/development.config.php.dist

```diff
@@ -1,19 +1,19 @@
 <?php
 /**
  * @license   http://opensource.org/licenses/BSD-3-Clause BSD-3-Clause
- * @copyright Copyright (c) 2014 Zend Technologies USA Inc. (http://www.zend.com)
+ * @copyright Copyright (c) 2014-2016 Zend Technologies USA Inc. (http://www.zend.com)
  */
 
-return array(
+return [
     // Development time modules
-    'modules' => array(
-        'ZFTool',
+    'modules' => [
+        'ZendDeveloperTools',
         'ZF\Apigility\Admin',
-    ),
+    ],
     // development time configuration globbing
-    'module_listener_options' => array(
-        'config_glob_paths' => array('config/autoload/{,*.}{global,local}-development.php'),
+    'module_listener_options' => [
+        'config_glob_paths' => ['config/autoload/{,*.}{global,local}-development.php'],
```

Comment:
Don't we need to have here also absolute paths as it is in `application.config.php` ?